### PR TITLE
ci: Reduce reviewers scope

### PR DESCRIPTION
**Pull request by jesse-c**
# What's changed

From a discussion in the product<>engineering retro today. The
tech-devs team is quite broad, and ends up with people receiving a lot
of emails.

The same as https://github.com/climatepolicyradar/navigator-admin-frontend/pull/84.

## Proposed version

Please select the option below that is most relevant from the list below. This
will be used to generate the next tag version name during auto-tagging.

- [x] Skip auto-tagging
- [ ] Patch
- [ ] Minor version
- [ ] Major version

Visit the [Semver website](https://semver.org/#summary) to understand the
difference between `MAJOR`, `MINOR`, and `PATCH` versions.

Notes:

- If none of these options are selected, auto-tagging will fail
- Where multiple options are selected, the most senior option ticked will be
  used -- e.g. Major > Minor > Patch
- If you are selecting the version in the list above using the textbox, make
  sure your selected option is marked `[x]` with no spaces in between the
  brackets and the `x`

### CODE REVIEW

**Comment by jesse-c** [comment]
@olaughter: Could you please add the team again, like you did for the other PR?

File: .github/CODEOWNERS
Comment (katybaulch) [inline .github/CODEOWNERS:15]: @jesse-c the software team exists, but it needs write access to this repo :)

Comment (olaughter) [reply .github/CODEOWNERS:15]: Heyo! I've just added software as write on this repo 👍

**Comment by olaughter** [comment]
> @olaughter: Could you please add the team again, like you did for the other PR?

Done! Sorry for being slow 😅

File: .github/CODEOWNERS
Comment (jesse-c) [reply .github/CODEOWNERS:15]: Haha, thank you!

**Comment by jesse-c** [comment]
@katybaulch: I've rebased it, and waiting for CI. I've requested a re-review :)

**Review by katybaulch — COMMENTED** [review]
amazing, thanks Jesse!!! 🙏🏻 ✨

**Review by katybaulch — APPROVED** [review]

**Pull request merged**
